spec/intro.dd

```diff
@@ -2,11 +2,13 @@ Ddoc
 
 $(SPEC_S Introduction,
 
-        $(P The D programming language is a general purpose systems programming language.
-        To that end, a D program is a collection of modules that can be compiled separately
-        to native code that is combined with libraries and compiled C code by a linker to create a
-        native executable.
-        )
+$(P D is a general-purpose systems programming language with a C-like syntax that compiles to native code. 
+It is strongly typed and supports both automatic (garbage collected) and manual memory management. 
+D programs are structured as modules that can be compiled separately and linked with external libraries 
+to create native libraries or executables.)
+
+$(P This document is the reference manual for the D Programming Language. For more information and 
+other documents, see [dlang.org](https://dlang.org/).)
```

Comment:
You can not use markdown here yet. You need to use Ddoc macros like LINK2

Reply:
Okay thanks - updated.